Microsoft.Toolkit.Uwp.SampleApp/SamplePages/ColorPickerButton/ColorPickerButtonXaml.bind

```diff
@@ -0,0 +1,122 @@
+﻿<Page
+    xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
+    xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
+    xmlns:controls="using:Microsoft.Toolkit.Uwp.UI.Controls"
+    xmlns:d="http://schemas.microsoft.com/expression/blend/2008"
+    xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006"
+    mc:Ignorable="d">
+
+  <Page.Resources>
+    <SolidColorBrush Color="{ThemeResource SystemChromeLowColor}" x:Key="SystemControlForegroundChromeLowBrush"/>
+  </Page.Resources>
+
+  <Grid>
+    <Grid.ColumnDefinitions>
+      <ColumnDefinition Width="*" />
+      <ColumnDefinition Width="*" />
+    </Grid.ColumnDefinitions>
+    <Grid.RowDefinitions>
+      <RowDefinition Height="*" />
+      <RowDefinition Height="*" />
+    </Grid.RowDefinitions>
+    <!-- Example 1 -->
+    <Border Grid.Column="0"
+            Grid.Row="0"
+            Background="{ThemeResource SystemChromeMediumColor}"
+            CornerRadius="4"
+            Height="100"
+            Width="300"
+            Padding="10">
+      <TextBlock TextAlignment="Center"
+                 VerticalAlignment="Center">
+        Box-shaped spectrum <LineBreak />
+        Alpha channel disabled
+      </TextBlock>
+    </Border>
+    <controls:ColorPickerButton x:Name="ColorPickerButton1"
```

Comment:
For this example, I noticed that if the color is a specific shade or very dark the palette selector at the bottom won't make a change:
![image](https://user-images.githubusercontent.com/24302614/88098031-f9145100-cb4d-11ea-9554-4d8226143089.png)

![image](https://user-images.githubusercontent.com/24302614/88098130-206b1e00-cb4e-11ea-9347-d97febaf5e5b.png)

I think this is because it's calculation is mapping to basically the same color. Should we somehow force a minimum step in value change for these different lower/higher 'buttons'?

It's also currently a destructive operation, if I keep going darker, I get stuck and can't go lighter, but even before then, I can lose the color that I was on.

Reply:
This is an open item that I'm actually hoping someone from Microsoft can help with. I won't really be able to solve it myself without a monumental amount of effort or a really good idea. From the future ToDo list: The mini selected color palette should follow the Windows accent color lighter/darker shades algorithm. However, the implementation of this algorithm is currently unknown.)

If I knew the algorithm used in windows to calculate accent colors I would just use that. What is in code now is a hack based on approximates: https://github.com/windows-toolkit/WindowsCommunityToolkit/issues/3363#issuecomment-650562951. Trying to design an algorithm that is perceptual, accounts for edge cases (black/white) and meets contrast requirements is no easy task.

Reply:
> It's also currently a destructive operation, if I keep going darker, I get stuck and can't go lighter, but even before then, I can lose the color that I was on.

I think this is always going to be a destructive operation to a degree. The only way to make it non-destructive would be a 'simple' algorithm that just varies the HSV value channel by a fixed percentage to create the accent colors. However, to account for all perceptual differences, saturation and even hue seem to require slight adjustments.

Reply:
I also noticed the value slide on the left was changing our selection of the color (unlike the built in picker). For instance if I start with green:

![image](https://user-images.githubusercontent.com/24302614/88101867-83ab7f00-cb53-11ea-9ea5-31f43d220dea.png)

But then slide the value slider:

![image](https://user-images.githubusercontent.com/24302614/88101914-96be4f00-cb53-11ea-8c06-28357f086987.png)

We've now lost our original color (see how in this one it's now near the top instead of on the green we originally picked). I see the quick palette preview at the bottom and the value slider on the left as quick shortcut helpers to modify the raw color, is that the actual case here in this implementation?

I think in both scenarios though, we should be leaving the picked raw color in the main area the same and allow the user to go back and forth. This should help with the above scenario as well as we'd be able to maintain the original intent?

Reply:
Really digging the design though, it's super slick btw. 🦙❤

You can see this same issue with the palette picker here too, if I start with Blue:

![image](https://user-images.githubusercontent.com/24302614/88102591-adb17100-cb54-11ea-94fa-c4180a20fc90.png)

And then modify the value with the palettizer below, I eventually just get stuck in black or grays, etc...

![image](https://user-images.githubusercontent.com/24302614/88102666-cc176c80-cb54-11ea-818f-c22eb075c409.png)